examples/example.rst

```diff
@@ -0,0 +1,23 @@
+Examples
+========
+
+This are example application of using `Flask-Reuploaded`. 
+
+.. 
+    _ TODO: Add example of serving by nginx
```

Comment:
Please remove that comment. This is out of scope of this documentation. Imagine Nginx changes the way it is configured, then we would need to update this documentation. You could link to a another tutorial or so, or to the official Nginx documentation.

Reply:
I'll remove this line becous I wrote it for myself :) 
What I mean is to add example of serving files using the UPLOADED_FILES_URL configuration. 

By this configuration, The extension is supposed to give valid file urls that can be served by servers like nginx. I suggest that we should add example for this scenario as the `configuration.rst` & `explanation.rst` sections that mention this feature are still vague in my opinion.